tests/integration/targets/netaddr_test/tasks/include/in_network.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "in_network: Check if 10.1.1.1 is in 10.0.0.0/8"
+  assert:
+    that: "{{ '10.1.1.1' is ansible.utils.in_network '10.0.0.0/8' }}"
+
+- name: "in_network: Check if 10.1.1.1 is not in 192.168.1.0/24"
+  assert:
+    that: "{{ '10.1.1.1' is not ansible.utils.in_network '192.168.1.0/24' }}"
+
+- name: "in_network: Test invalidness"
+  assert:
+    that: "{{ '8.8.8.8' is ansible.utils.in_network '10.0.0.0/8' }}"
+  ignore_errors: true
```

Comment:
Same as above. If is it to check failed condition you can use set_fact to set a variable and later assert of the variable is `False`